app/controllers/admin/delete_collection_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Admin
+  # Controller for deleting a Collection as an administrator
+  class DeleteCollectionController < Admin::ApplicationController
+    before_action :set_collection
+
+    def new
+      authorize!
+
+      render :form
+    end
+
+    def destroy
+      authorize!
+
+      collection.destroy!
+      render_delete_success
+    end
+
+    attr_reader :collection_form, :collection
+
+    private
+
+    def druid
+      params[:druid] || params[:collection_druid]
```

Comment:
Are you sure you need both of these?

Reply:
Nah - druid was coming from the route, updated that to collection_druid so only need the one now. :+1+